docs/general/events.md

```diff
@@ -3,13 +3,35 @@ linkTitle: Events
 aliases: [docs/specs/semconv/general/events-general]
 --->
 
-# Semantic Conventions for Event Attributes
+# Semantic Conventions for Events
 
 **Status**: [Experimental][DocumentStatus]
 
-This document describes the attributes of standalone Events that are represented
+This document describes the characteristics of standalone Events that are represented
 in the data model by `LogRecord`s.
 
+Semantically, an Event is a named occurrence at an instant in time. It signals that
+"this thing happened at this time" and provides additional specifics about the occurrence.
+Examples of Events might include things like uncaught exceptions, button clicks, user logout,
+network connection severed, etc.
+
+In OpenTelemetry, Events are implemented as a specific type of `LogRecord` that conforms to
+the conventions included here, and Events
+[have their own API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md).
+The API abstracts away knowledge of `LogRecord` so that users only deal with Event
+semantics.
+
+In addition to a required name, an Event may contain a _payload_ of any type permitted by the
```

Comment:
Well actually been defining that the _payload_ is a Map<string, Attribute> values rather than `of any type` as the Event Api is being defined as receiving Attributes whichit will then place into the `body`. Likewise with the prototyping of the build-tools...

Reply:
That's fine. I'm not sure that's incompatible with the vernacular here, is it? 

This was trying to just say that the payload is whatever is allowed by the body. If you brought this up because you think it needs to be more specific here, can we do that as a more precise follow-up PR?